src/maze_bench_nim.nim

```diff
@@ -8,48 +8,51 @@ import
   point,
   mazesolver
 
-let image = readImage("maze1000.png")
-echo fmt"Loaded image: {image.width} x {image.height}"
-
-echo "Transforming image into maze..."
-var now = getMonoTime()
-
-let maze = newSeq2D[bool](image.width, image.height)
-
-var
-  column: int
-  row: int
-for i, rgba in image.data:
-  maze[column, row] = (rgba.r + rgba.g + rgba.b) > 0
-  if column + 1 >= image.width:
-    column = 0
-    row += 1
+proc main() =
```

Comment:
Adding a `main` function actually did speed it up to my surprise; why is this? I thought there'd be no difference

Reply:
Actually tried to dig up some information on this, but couldn't really find anything. It might have to do with global variables, but I don't really see why those would affect performance this much.